.github/workflows/automation-load-tests.yml

```diff
@@ -112,5 +112,4 @@ jobs:
           basic-auth: ${{ secrets.GRAFANA_CLOUD_BASIC_AUTH }}
           hostname: ${{ secrets.GRAFANA_CLOUD_HOST }}
           this-job-name: Automation Load Test
-          test-results-file: '{"testType":"go","filePath":"/tmp/gotest.log"}'
```

Comment:
Is it okay that this is still referenced in the ondemand and nightly tests?

Reply:
yes those should be fine. For both of those tests, we have these files since the test runs on the runner itself and not on remote runner.